### PR TITLE
fix pip-freeze dependencies

### PR DESCRIPTION
**Pull request by rccrdpccl**
pip-freeze has pip-compile as dependency, and it was broken. Pip-tools 6.13.0 or greater was required leading to a loop (unable to freeze dependency because pip-freeze wouldn't work).

This PR fixes this issue.

### CODE REVIEW

File: Dockerfile.assisted-installer-deployment [hunk 1]
```diff
@@ -7,6 +7,7 @@ RUN dnf install -y \
     make \
     skopeo \
     libffi-devel \
+    krb5-devel \
```
Comment (rccrdpccl) [inline Dockerfile.assisted-installer-deployment:10]: This was a requirement from pip-compile, not sure why though

Comment (osherdp) [reply Dockerfile.assisted-installer-deployment:10]: what do you see if it's not present?

Comment (rccrdpccl) [reply Dockerfile.assisted-installer-deployment:10]: > what do you see if it's not present?

Error when executing pip-compile (executed by make pip-freeze): `krb5-config: command not found`

Comment (rccrdpccl) [reply Dockerfile.assisted-installer-deployment:10]: I think it comes from pip-compile, but I was also trying to add another unrelated dependency. I'll make sure it comes from pip-compile

Comment (rccrdpccl) [reply Dockerfile.assisted-installer-deployment]: whoops, it was indeed coming from the other dependency. Mystery solved :sweat_smile:

**Review by osherdp — APPROVED** [review]

**Pull request closed without merging**